pytest.ini

```diff
@@ -0,0 +1,33 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+[pytest]
+markers =
+    seed: set the python, numpy and mxnet random seeds to a specified value for test reproducibility
+    serial: mark a test that requires more resources to run that are thus only suitable for serial run.
+    remote_required: mark a test that requires internet access.
+    gpu: mark a test that requires GPU.
+    integration: mark an integration test
+    onnx_coverage: ONNX coverage test
+    garbage_expected: this test leaks ndarray references. The tested functionality is broken or there is a Python bug.
+    integrationtest_onnx: mark a test for onnx integration test in CI
```

Comment:
how is it different from regular integration test?

Reply:
Due to the heavy computation load of onnx tests, we only run the full onnx tests in nightly CI, and run a subset of it in the PR integration tests. This marker is used to mark the subset of onnx tests. I can change the tag to something like `onnx_pr_test` if it's more clear.  wdyt?

Reply:
since you are running the ONNX tests one file at a time, I think using the integration mark should suffice.

Reply:
Makes sense. Switched to `integration` tag